spec/bson/byte_buffer_spec.rb

```diff
@@ -30,16 +30,62 @@
       let(:buffer) do
         described_class.new
       end
-
-      before do
-        buffer.put_int32(5)
+      context '#put_int32' do 
+        before do
+          buffer.put_int32(5)
+        end
+
+        it 'returns the length of the buffer' do
+          expect(buffer.length).to eq(4)
+        end
```

Comment:
Correct me if I'm wrong here but this looks like a duplicate of the test you have below?

Reply:
The one right below it is put_uint32 (unsigned) while this one is put_int32 (signed)